config/config.go

```diff
@@ -57,25 +69,63 @@ func FindConfig(fs afero.Fs, configFile string) ([]byte, int, error) {
 	return b, v, nil
 }
 
+func findConfig(fs afero.Fs, configFile string) ([]byte, int, error) {
```

Comment:
i'm probably missing something here, but are `findJSON` and `findConfig` different?